setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='testimony',
-    version='1.0.3',
+    version='1.0.4',
```

Comment:
Its been a while since we started accepting the first line of docstring as the test case name which went in 1.0.3.  This current PR is just an extension of the same but not really worthy of a new minor version.